<launch>
    <node pkg="move_arm" type="move_arm_simple_action" name="youbot_move_arm">
        <param name="group" type="string" value="arm" />
        <remap from="arm_ik" to="youbot_arm_kinematics/get_constraint_aware_ik" />
        <param name="controller_action_name" type="string" value="arm_controller/follow_joint_trajectory" />
    </node>
</launch>
